video/out/vo_kitty.c

```diff
@@ -114,6 +121,35 @@ static struct sigaction saved_sigaction = {0};
 static bool resized;
 #endif
 
+static inline void write_bstr_passthrough(struct priv *p, bstr bs)
```

Comment:
Do we really need to append those prefixes in every append and on write too? Seems like there will be redundancy in final string.

Reply:
this function is only used to passthrough the `write_bstr(KITTY_ESC_DELETE_ALL)` call in `reconfig`
other writes use the normal `write_bstr`/`write_str` functions, so there's no redundancy

Reply:
since this is only used once i can extract this out where it is used if you want

Reply:
it's fine as is